shared_model/interfaces/common_objects/common_objects_factory.hpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_COMMON_OBJECTS_FACTORY_HPP
+#define IROHA_COMMON_OBJECTS_FACTORY_HPP
+
+#include <memory>
+
+#include "common/result.hpp"
+#include "interfaces/common_objects/account.hpp"
+#include "interfaces/common_objects/peer.hpp"
+#include "interfaces/common_objects/types.hpp"
+#include "interfaces/common_objects/domain.hpp"
+
+namespace shared_model {
+  namespace interface {
+    /**
+     * CommonObjectsFactory provides methods to construct simple objects
```

Comment:
What is "simple object"? Can you, please, define it in more specific way?

Reply:
I guess "common_objects" is a better name here since they are in the folder with this name